.evg.yml

```diff
@@ -94,8 +96,11 @@ tasks:
         params: 
           script: |
             set -e
-            cd stitch-ios-sdk/StitchCore
-            swiftlint
+            pushd SwiftLint
+            export PATH=`pwd`:$PATH
```

Comment:
small warning that paths with whitespace may get messed up by this, consider 2-step process

Reply:
We use this pattern throughout `.evg.yml` because we know the paths won't have whitespace, so I'm going to leave as is.